test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/Binders/SimpleTypeModelBinderTest.cs

```diff
@@ -350,112 +350,6 @@ public static TheoryData<Type> BiggerNumericTypes
             Assert.Null(error.Exception);
         }
 
-        [Fact]
```

Comment:
We shouldn't remove these. `SimpleTypeModelBinder` still supports `enum` types. And, users can remove the new `EnumTypeModelBinderProvider` from `MvcOptions.ModelBinderProviders` instead of changing the new `AllowBindingUndefinedValueToEnumType` property.